utils/bq_utils.py

```diff
@@ -362,6 +362,6 @@ def get_bq_data_for_inference(ecosystem, day_count, date_range) -> pd.DataFrame:
         sys.exit(0)
 
     _logger.info("Creating description column for NLP")
-    df["description"] = df["title"].fillna(value="").map(str) + " " + df["body"].fillna(value="")
+    df["description"] = df["title"].fillna(value="").map(str) + " " + df["body"].fillna(value="").map(str)
```

Comment:
In pandas it is better to deal with data at a column level rather than a row level. Apply Fillna for title column separately, body separately and then apply the function here.

Reply:
done